packages/react-router-redux/modules/ConnectedRouter.js

```diff
@@ -1,3 +1,4 @@
+import invariant from 'invariant'
```

Comment:
Oops! Interesting that the test didn't fail locally with that missing. Fix incoming.